### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by steviebeenz**
!wip

### CODE REVIEW

File: src/main/frontend/src/components/entrypoints/users/Notifications.vue [hunk 1]
```diff
@@ -159,8 +159,8 @@ export default {
         };
     },
     mounted() {
-        const invites = $('.invite-content');
-        invites.css('height', invites.width());
+        const invites = document.querySelector('.invite-content');
+        invites.style.height = invites.style.width;
```
Comment (Machine-Maker) [inline src/main/frontend/src/components/entrypoints/users/Notifications.vue:163]: this is **not** the same thing. The element has no width. The jquery width method calculates the actual width of the element. What you changed it to is what I originally had, but it's not the same, and won't work.

File: src/main/frontend/src/components/entrypoints/versions/CreateVersion.vue [hunk 2]
```diff
@@ -370,7 +370,7 @@ export default {
             alert.find('.file-size').text(filesize(files[0].size));
             alert.fadeIn('slow');
 
-            $('#form-url-upload').css('display', 'none');
+            document.querySelector('form-url-upload').style.display = "none";
```
Comment (Machine-Maker) [inline src/main/frontend/src/components/entrypoints/versions/CreateVersion.vue:373]: I think this needs to have a `#` before it right?

**Review by Machine-Maker — CHANGES_REQUESTED** [review]

File: src/main/frontend/src/components/entrypoints/users/Notifications.vue (hunk 1 above)
Comment (steviebeenz) [reply src/main/frontend/src/components/entrypoints/users/Notifications.vue:163]: Thats from before testing. Will work our another way of doing this WITHOUT JQUERY

File: src/main/frontend/src/components/entrypoints/versions/CreateVersion.vue (hunk 2 above)
Comment (steviebeenz) [reply src/main/frontend/src/components/entrypoints/versions/CreateVersion.vue:373]: No it needs to be a getElementById. Thanks though

**Comment by steviebeenz** [comment]
ZappFix Head Dump

      console.error("ZAPPFIX: Fixed Empty Property (Array) (REMOVE WHEN FIXED)");
      window.COLORS = [];
      console.error("ZAPPFIX: Fixed Empty Property (STRING) (REMOVE WHEN FIXED)");
      window.DiscourseEmbed = {};
      window.DiscourseEmbed.discourseUrl = ""
      console.error("ZAPPFIX: Fixed Empty Property (Array) (REMOVE WHEN FIXED)");
      window.CHANNELS = []
      console.error("ZAPPFIX: Fixed Empty Property (OBJECT) (REMOVE WHEN FIXED)");
      window.csrfInfo = {}
      console.error("ZAPPFIX: Fixed Empty Property (STRING) (REMOVE WHEN FIXED)");
      window.csrfInfo.headerName = ""

File: src/main/frontend/src/components/entrypoints/users/Notifications.vue (hunk 1 above)
Comment (steviebeenz) [reply src/main/frontend/src/components/entrypoints/users/Notifications.vue:163]: It is .offsetWidth for reference.

**Comment by steviebeenz** [comment]
@Machine-Maker It's all resolved (I hope). Thanks for the suggestions.

**Comment by steviebeenz** [comment]
Ok so here I will go into depth about what this PR is. It is some Jquery Removal, and some compression (+ ZAPPFIX to temporarily patch some bugs) .

File: src/main/frontend/.postcssrc.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 module.exports = {
     plugins: {
         autoprefixer: true,
+        cssnano: true
```
Comment (MiniDigger) [inline src/main/frontend/.postcssrc.js:4]: our css is already minimized, remove this

File: src/main/frontend/public/index.html [hunk 4]
```diff
@@ -6,6 +6,21 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <!-- START ZAPPFIX BETA INJECT -->
```
Comment (MiniDigger) [inline src/main/frontend/public/index.html:9]: wtf is this?

File: src/main/frontend/vue.config.js [hunk 5]
```diff
@@ -18,6 +19,8 @@ module.exports = {
     configureWebpack: {
         plugins: [new StatsPlugin('stats.json')],
         optimization: {
+            minimize: true,
```
Comment (MiniDigger) [inline src/main/frontend/vue.config.js:22]: our js is already minimzed

**Review by MiniDigger — CHANGES_REQUESTED** [review]
yeah, Idk abotu this

**Comment by kashike** [comment]
Yea, I am going to close this pull request. Please actually look at what is already implemented before working on something, and keep the irrelevant and useless changes out of pull requests.

**Pull request closed without merging**